Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -78,7 +78,7 @@ public void OnClickSave()
         this.CloseDialog();
         WorldController.Instance.spawnInventoryController.SetUIVisibility(developerModeToggle.isOn);
         LocalizationTable.SetLocalization(languageDropdown.value);
-        //FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
+        ///FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
```

Comment:
this should probably just be a blank space, or deleted since git tracks lines like this